css/styles.css

```diff
@@ -0,0 +1,26 @@
+*{
+    color: white;
+    background-color: green;
+} /* this is universal selector */
+
+
+/* below are type selectors */
+div{
+    color: red;
+}
+
+h3{
```

Comment:
Use correct css styling- Css should apply two spaces (You have used 4 spaces)

Reply:
- [ ] Always make sure to include an explanation, a "why", for each requested change, and use polite words like `Kindly`, and `It would be better`. Kindly Implement it. 👍

Reply:
@roshan-bajgain , Please note i have updated by request above

Reply:
Hi, @kiboma2021 !

I notice that you applied the requested changes and edited your original comment. Great job! :+1: